README.md

```diff
@@ -1125,6 +1125,32 @@ Note that each user request and response from the agent will be in their own tra
 
 **Troubleshooting:** If traces don't appear, verify `OTEL_EXPORTER_OTLP_ENDPOINT` is set on deployments and check service logs for OpenTelemetry initialization messages
 
+#### Viewing Traces in OpenShift Console
```

Comment:
Should the topic above called Viewing Traces be changed to something like Viewing Traces with Jaeger ?